Source/Core/Core.swift

```diff
@@ -740,6 +756,126 @@ extension FormViewController : UITableViewDelegate {
         }
         return view.bounds.height
     }
+    
+    open func tableView(_ tableView: UITableView, canEditRowAt indexPath: IndexPath) -> Bool {
+        guard let section = form[indexPath.section] as? MultivaluedSection else { return false }
+        let row = form[indexPath]
+        guard !row.isDisabled else { return false }
+        guard !(indexPath.row == section.count - 1 && section.multivaluedOptions.contains(.Insert) && section.showInsertIconInAddButton) else {
+            return true
+        }
+        if indexPath.row > 0 && section[indexPath.row - 1] is BaseInlineRowType && section[indexPath.row - 1]._inlineRow != nil {
+            return false
+        }
+        return true
+    }
+    
+    public func tableView(_ tableView: UITableView, commit editingStyle: UITableViewCellEditingStyle, forRowAt indexPath: IndexPath) {
+        if editingStyle == .delete {
+            let row = form[indexPath]
+            var section = row.section!
+            if let _ = row.baseCell.findFirstResponder() {
+                tableView.endEditing(true)
+            }
+            section.remove(at: indexPath.row)
+            DispatchQueue.main.asyncAfter(deadline: .now()) {
```

Comment:
Originally i was adding some delay ( DispatchQueue.main.asyncAfter(deadline: .now() + nanoseconds(xyz))) but then i realized there were no need for that block execution dalay.

If it is exactly the same i would also like to use `DispatchQueue.main.async`.